firebase-crashlytics/src/androidTest/java/com/google/firebase/crashlytics/internal/CrashlyticsTestCase.java

```diff
@@ -21,6 +21,8 @@
 
 /** Base class for all our TestCases. */
 public class CrashlyticsTestCase extends TestCase {
+  // TODO(mrober): Setup the workers with test executors in here?
```

Comment:
This is something we want to do next?

Reply:
Discussed offline. We want to have more flexibility in individual test cases, then some general test structure that does things for you behind the scene. So we won't do this.